plugins/catalog-node/package.json

```diff
@@ -40,6 +40,7 @@
     "@backstage/catalog-client": "workspace:^",
     "@backstage/catalog-model": "workspace:^",
     "@backstage/errors": "workspace:^",
+    "@backstage/plugin-catalog-backend": "workspace:^",
```

Comment:
This is not possible - it makes a circular dependency. The type needs to be deprecated in the backend and introduced in -node instead.

Reply:
Updated.